tests/integration/targets/ipam_host/meta/main.yml

```
@@ -0,0 +1,4 @@
+---
+dependencies:
```

Make Dependencies a list instead. 
```suggestion
dependencies: [ setup_ip_space , setup_subnet ]
```

---

modified